src/main/kotlin/ucfs/claimant/consumer/properties/AwsProperties.kt

```diff
@@ -0,0 +1,16 @@
+package ucfs.claimant.consumer.properties
+
+import org.springframework.boot.context.properties.ConfigurationProperties
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+
+@Configuration
+@ConfigurationProperties(prefix = "aws")
+data class AwsProperties(var saltParameterName: String = "/ucfs/claimant-api/nino/salt",
+                         var cmkAlias: String = "alias/ucfs_etl_cmk") {
```

Comment:
I don't think the parameter name or CMK alias should be here, I don't think they should have a default value

Reply:
Removed the defaults and updated the docs.